src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -25,12 +26,16 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             placeholder="Message prompt"
             keyboardAppearDelay={500}
             editable={!props.isOtherProfile}
+            maxLength={350}
           />
           <Text style={styles.description}>{props.value ? props.value.length : 0}/350</Text>
           {props.error ? <Text style={styles.errorText}>{props.error}</Text> : null}
         </View>
         {!props.isOtherProfile && (
-          <Button style={styles.button} textStyling={styles.textStyling} onPress={props.handleSave}>
+          <Button
+            btnStyle={styles.button}
+            textStyling={styles.textStyling}
+            onPress={props.handleSave}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a `maxLength` prop to the `AutoFocusTextArea` component is a good practice to prevent users from entering more characters than the system can handle. However, it would be better if the maximum length (350 in this case) was not hardcoded and instead passed as a prop or defined as a constant. This would make the code more maintainable and flexible for future changes.

```diff
- maxLength={350}
+ maxLength={props.maxLength || DEFAULT_MAX_LENGTH}
```

In the above code, `DEFAULT_MAX_LENGTH` should be a constant defined in an appropriate place in your codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->